android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -1288,9 +1299,11 @@ public void onPermissionRequest(final PermissionRequest request) {
         } else if (requestedResource.equals(PermissionRequest.RESOURCE_VIDEO_CAPTURE)) {
           androidPermission = Manifest.permission.CAMERA;
         } else if(requestedResource.equals(PermissionRequest.RESOURCE_PROTECTED_MEDIA_ID)) {
-          androidPermission = PermissionRequest.RESOURCE_PROTECTED_MEDIA_ID;
+          if (mAllowsProtectedMedia) {
+              grantedPermissions.add(requestedResource);
+          }
```

Comment:
Do you think it would hurt to keep the previous code if `mAllowsProtectedMedia` is `false`, just in case there was some scenario (e.g. certain Android/WebView version??) where it actually did something useful?

Reply:
A downside I see to doing that would be that it might hurt the code readability, and result in undocumented/unexpected behavior for developers ending up that case.

Now, I don't think it could do any harm since it would ask for the end-user approval anyway.

That being said, I'm gonna try to make the change this afternoon, I'll let you know once it's done.